Source/Scene/Camera.js

```diff
@@ -177,6 +177,12 @@ function Camera(scene) {
    * @default 100000.0;
    */
   this.defaultZoomAmount = 100000.0;
+  /**
+   * If set, the camera will not be able to tilt past this angle, expressed in radians.
+   * @type {Number}
+   * @default undefined
+   */
+  this.maximumTilt = undefined;
```

Comment:
Similar to `maximumZoomAmount` and `minimumZoomAmount`, any interaction related limitations are better suited for the `ScreenSpaceCameraController`.